.jenkins

```diff
@@ -340,6 +340,41 @@ pipeline {
                         }
                     }
                 }
+                stage('HIP-ROCm-6.0') {
```

Comment:
Do you anything else in the name of the build to indicate that this one, in contrast to the other that set hipcc as the cxx compiler, uses amdclang directly?